extensions/websockets/server/deployment/src/main/java/io/quarkus/websockets/deployment/ServerWebSocketProcessor.java

```diff
@@ -54,6 +71,19 @@ void scanForAnnotatedEndpoints(CombinedIndexBuildItem indexBuildItem,
         }
     }
 
+    @BuildStep
+    void scanForCodecs(CombinedIndexBuildItem index,
+            BuildProducer<ReflectiveHierarchyBuildItem> reflectiveHierarchyBuildItemProducer) {
+        CODECS.stream().forEach(
+                codec -> index.getIndex().getAllKnownImplementors(codec).stream()
+                        .filter(implementor -> !Modifier.isAbstract(implementor.flags()))
+                        .forEach(implementor -> JandexUtil.resolveTypeParameters(
+                                implementor.name(),
+                                codec, index.getComputingIndex()).forEach(
```

Comment:
@geoand I have a doubt here ...
getComputingIndex or getIndex ?
It's working just like this, but I don't know the difference ...

Reply:
`getIndex` is the correct way to do it

Reply:
I'll fix this and resquash it right now !

Reply:
@geoand 
If I use getIndex I get this error : 
	[error]: Build step io.quarkus.websockets.deployment.ServerWebSocketProcessor#scanForCodecs threw an exception: java.lang.IllegalArgumentException: The following classes were not part of the index and could be the reason that the captured generic type of 'jakarta.websocket.Decoder$Text' could not be determined: [jakarta.websocket.Decoder$Text]
	
So maybe getCombinedIndex is the only way ...

Reply:
💪🏼